<?xml version="1.0"?>
<launch>

  <group ns="gnss_observations">
    <arg name="ws_path" default="$(find gnss_observations)"/>
    <arg name="opt_file" default="$(find gnss_observations)/config/rtkrcv.conf"/>
    <arg name="opt_ref_file" default="$(find gnss_observations)/config/rtkrcv_ref.conf"/>

    <node pkg="gnss_observations" type="gnss_data" name="gnss_data">
    <!-- node parameters -->
      <param name="ws_path" type="string" value="$(arg ws_path)"/>  
      <param name="opt_file" type="string" value="$(arg opt_file)"/>  
      <param name="opt_ref_file" type="string" value="$(arg opt_ref_file)"/>  
      <param name="start_script" type="string" value="$(arg ws_path)/gnssstart.sh"/>  
      <param name="post_proc" type="bool" value="false"/>  
    </node>
  </group>

</launch>
